config/routes.rb

```diff
@@ -1,12 +1,14 @@
 # Draw all of the routes that will be used in Hubstats
 Hubstats::Engine.routes.draw do
-  root to: "pull_requests#index" # sets default root to be the pulls page
   post "/handler" => "events#handler", :as => :handler
-  resources :deploys, :only => [:create, :index, :show] # routes to index, show, and to create method
-  get "/metrics" => "repos#dashboard", :as => :metrics # routes to list of repos and stats
+  root to: "pull_requests#index" # sets default root to be the pulls page
   get "/pulls" => "pull_requests#index", :as => :pulls # routes to list of pulls
+  resources :deploys, :only => [:create, :index, :show] # routes to index, show, and to create method
+  get "/teams" => "teams#index", :as => :teams # routes to the list of teams and stats
```

Comment:
Teams is a resource following Rails RESTful convetions so you can simplify this to:
```
resources :teams, :only => [:index, :show]
```